tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -468,8 +468,8 @@
         return create_context_action("create_virtual_pressure_source", context, {source_type, metadata});
     };
 
-    window.test_driver_internal.update_virtual_pressure_source = function(source_type, sample, context=null) {
-        return create_context_action("update_virtual_pressure_source", context, {source_type, sample});
+    window.test_driver_internal.update_virtual_pressure_source = function(source_type, sample, estimate = -1.0, context=null) {
```

Comment:
Is -1.0 the right default value here? I couldn't tell from the spec why that would be the case.

Reply:
Agree! I removed it and checked it still works when rolling out in chromium.

Thanks